examples/TID_model_custom_property/Modelling_TID_TNID_example_notebook.ipynb

```diff
@@ -0,0 +1,471 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    cfg.sim.activity_timestep = 10.0 # update frequency of the constraint function. 
</pre><p>I don't think this parameter is relevant for this notebook?</p><p> </p><p>Probably for this example you could just avoid a custom cfg alltogether (start date doesn't really matter either, does it?) just to keep it simple and on the point</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
I would use the starting date as it is because it is the starting date that I used in SPENVIS to calculate the total dose. The charge that the spacecraft takes depends on the position of the spacecraft wrt the sun and the solar activity. I can simplify the notebook by inserting directly the ephemerides at t=start and adding a comment on the matter above.